MozillaFoundation/foundation.mozilla.org
# 11177 donate help page notice

# Description
Link to sample test page: https://donate-foundation-s-11177-dona-fjocje.mofostaging.net/en/help/
Related PRs/issues: #11177 

This PR introduces "delayed response notice" at the top of the donate help page by adding the following to the code base:
- A new html template for the delayed response notice section
- A new boolean field for the `DonateHelpPage` titled `show_notice`. The delayed response notice will render based on whether or not this field is checked.
- Migrations needed for the new page field

# Screenshots

**`show_notice` is checked:**
![Screenshot 2023-09-29 at 15-56-47 Donate Help](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/6dfc5d9d-fcb2-4d63-9d74-ac52adfc419b)


**show_notice is unchecked:**
![Screenshot 2023-09-29 at 15-57-10 Donate Help](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/3cbd19e0-3513-45f1-87e7-ed4ec9c5d922)


# Steps to test

1. Visit the test page [here](https://donate-foundation-s-11177-dona-fjocje.mofostaging.net/en/help/)
2. Note that the delayed response notice is being rendered
3. Log into [the CMS ](https://foundation-s-11177-dona-fjocje.herokuapp.com, edit the page, and unselect the checkbox for `show_notice`
4. Visit the page again, and note that the notice is no longer being rendered.
5. If everything else is looking as expected, testing is complete!
